api/v1beta1/helmrepository_types.go

```diff
@@ -30,6 +30,11 @@ const (
 	// HelmRepositoryURLIndexKey is the key to use for indexing HelmRepository
 	// resources by their HelmRepositorySpec.URL.
 	HelmRepositoryURLIndexKey = ".metadata.helmRepositoryURL"
+	// HelmRepositoryTypeDefault is the default HelmRepository type.
```

Comment:
This API should not require changes.

Reply:
I was thinking we should have this backward compatible.

Reply:
The v1beta1 API is out of use, and no reconciler is processing this API version anymore. The API server automatically makes v1beta1 resources v1beta2.